src/test/java/picard/nio/PicardBucketUtilsTest.java

```diff
@@ -0,0 +1,21 @@
+package picard.nio;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.util.GCloudTestUtils;
+
+public class PicardBucketUtilsTest {
+
+    // Check that the extension scheme (e.g. "txt" vs ".txt" as the second argument) is consistent for cloud and local files
```

Comment:
I think this comment is obsolete since there is no longer a "txt" vs ".txt" issue (and also the reference to the "second argument" is a bit ambiguous given that there are two overloads being used here). Maybe:
```suggestion
    // Check that the extension scheme is consistent for cloud and local files
```

Reply:
done